### PR TITLE
Fix: System config value is not updated after save

### PR DESCRIPTION
**Pull request by mitesh-toptal**
The purpose of this PR is to fix system configuration value is not updated after save.

### CODE REVIEW

File: Plugin/ConfigPlugin.php [hunk 1]
```diff
@@ -53,22 +53,16 @@ public function aroundSave(
...
-            }
-        }
-        if (isset($data['groups']['settings']['fields']['enabled']['value'])) {
-            if ($data['groups']['settings']['fields']['enabled']['value'] == 1) {
-
+            } elseif ($data['groups']['settings']['fields']['enabled']['value'] == 1) {
                 if (isset($data['groups']['settings']['fields']['imgix_api_key'])) {
                     $apiKey = $data['groups']['settings']['fields']['imgix_api_key']['value'];
-
+
```
Comment (sherwinski) [inline Plugin/ConfigPlugin.php:63]:
```suggestion

```

File: Plugin/ConfigPlugin.php [hunk 2]
```diff
@@ -53,22 +53,16 @@ public function aroundSave(
...
-            if ($data['groups']['settings']['fields']['enabled']['value'] == 1) {
-
+            } elseif ($data['groups']['settings']['fields']['enabled']['value'] == 1) {
                 if (isset($data['groups']['settings']['fields']['imgix_api_key'])) {
                     $apiKey = $data['groups']['settings']['fields']['imgix_api_key']['value'];
-
+        
                     $apiKeyValidation = $this->helperData->getImgixApiKeyValidation($apiKey);
-
+
```
Comment (sherwinski) [inline Plugin/ConfigPlugin.php:65]:
```suggestion

```

**Review by sherwinski — COMMENTED** [review]
@mitesh-toptal 
What is the difference between using `isset` and ` == 1` here?

**Comment by mitesh-toptal** [comment]
@sherwinski 
> @mitesh-toptal What is the difference between using `isset` and ` == 1` here?

`isset()` function determines if the system configuration section fields are declared and its value is not equal to NULL.
`==1` check imgix the "Enabled" field is enabled or not.

**Review by sherwinski — APPROVED** [review]

**Pull request merged**